src/libs/actions/IOU.ts

```diff
@@ -1707,17 +1707,20 @@ function getDeleteTrackExpenseInformation(
     }
 
     if (actionableWhisperReportActionID) {
-        failureData.push({
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport?.reportID}`,
-            value: {
-                [actionableWhisperReportActionID]: {
-                    originalMessage: {
-                        resolution: null,
+        const actionableWhisperReportAction = ReportActionsUtils.getReportAction(chatReportID, actionableWhisperReportActionID);
+        if (ReportActionsUtils.isActionableTrackExpense(actionableWhisperReportAction)) {
```

Comment:
This condition is to make sure there is no type error when accessing ```resolution``` prop.

Reply:
@truph01 Got it. In that case, let us define `resolution` something like this and update the `originalMessage`. This way is neater. Otherwise, it will look like we want to push into `failureData` only if we have a valid Whisper Action which is not the case. We want to push into `failureData` with resolution value of `null` even if we do not get a valid Whisper Action. Does this make sense?

`const resolution = ReportActionsUtils.isActionableTrackExpense(actionableWhisperReportAction) ? ReportActionsUtils.getOriginalMessage(actionableWhisperReportAction)?.resolution ?? null : null;`